_posts/2021-01-12-updating-foreman-inventory-with-system-facts.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: post
+title: Updating Foreman inventory with system facts
+date: 2021-01-12 17:00:00
+author: Marek Hulan
+tags:
+- foreman
+- community
+- tutorial
+- howto
+- beginners
+---
+
+This is a continuation of [How to start with Foreman post](https://theforeman.org/2020/12/how-to-start-with-foreman.html).
+Today we'll look at how the inventory can be kept up to date by receiving facts from hosts regularly. Then we'll take a look
+where facts can be used. We'll illustrate the configuration on one config management system, but the same concept is applicable
+to other facts sources. Links to the related documentation will be provided.
+
+<!--more-->
+
+## Automatically update the inventory from configuration management
+
+Having to keep a big inventory up to date manually would be a lot of work. Foreman
+can receive updates from other tools used to manage the infrastructure and reflect
+them in the database. Also new Hosts can be created this way. If you use some
+configuration management system or you use a subscription manager (mostly RPM
```

Comment:
`a subscription manager` here is a bit odd since thats the client tools name in the case of `subscription-manager` and there aren't really others you could go use. I would opt for just saying `use subscription-manager`